reco_utils/recommender/sar/sar_singlenode.py

```diff
@@ -323,11 +329,96 @@ def recommend_k_items(self, test, top_k=10, sort_top_k=False, remove_seen=False)
         # drop invalid items
         return df.replace(-np.inf, np.nan).dropna()
 
+    def recommend_popular_items(self, top_k=10, sort_top_k=False):
+        test_users = pd.Series([0])
+        test_scores = np.array([self.item_frequencies])
+
+        # ensure we're working with a dense matrix
+        if isinstance(test_scores, sparse.spmatrix):
+            test_scores = test_scores.todense()
+
+        return self._get_top_k_items(
+            users=test_users, scores=test_scores, top_k=top_k, sort_top_k=sort_top_k
+        )
+
+    def recommend_similar_items(self, items, top_k=10, sort_top_k=False):
+        """Recommend top K items similar to the provided seed items
```

Comment:
Maybe be a bit more specific and say "k most similar items in terms of co-occurrence" or something along these lines.

Reply:
ok, will do